### PR TITLE
Fix-some-python-2-prints-in-nba-and-others

### PR DESCRIPTION
**Pull request by Paulnkk**
## Description
Please include a short summary of the change.
I fixed some python 2 prints in the nba example and the docscrape file.
Issue link (if applicable):
Found issue here: https://github.com/cvxpy/cvxpy/issues/2095

## Type of change
- [ ] New feature (backwards compatible)
- [ ] New feature (breaking API changes)
- [ ] Bug fix
- [x] Other (Documentation, CI, ...)

## [Contribution checklist](https://www.cvxpy.org/contributing/index.html#contribution-checklist)
- [ ] Add our license to new files.
- [x] Check that your code adheres to our coding style.
- [ ] Write unittests.
- [ ] Run the unittests and check that they’re passing.
- [ ] Run the benchmarks to make sure your change doesn’t introduce a regression.

### CODE REVIEW

**Comment by Paulnkk** [comment]
@phschiele I get an error with the 'gamestats' directory since it does not exist (or I can not find this directory)

**Comment by Paulnkk** [comment]
How should I proceed with this PR since another contributor made the same PR, and I am not sure if the gamestats directory exists, which is relevant to execute the print operations?

File: doc/sphinxext/docscrape.py [hunk 1]
```diff
@@ -469,7 +469,7 @@ def __str__(self):
 
         if self._role:
             if not roles.has_key(self._role):
-                print "Warning: invalid role %s" % self._role
+                print(f"Warning: invalid role {self._role}")
```
Comment (phschiele) [inline doc/sphinxext/docscrape.py:472]: @SteveDiamond Do you know what this file is doing? Appears to be related to making docstrings formatted according to numpy conventions.

Maybe we can get rid of the files in `doc/sphinxext/` alltogether.

Comment (SteveDiamond) [reply doc/sphinxext/docscrape.py:472]: I don't know what it is, sorry.

Comment (SteveDiamond) [reply doc/sphinxext/docscrape.py:472]: If the example is not runnable we should probably remove it.

Comment (phschiele) [reply doc/sphinxext/docscrape.py:472]: @Paulnkk I agree, you can remove the file in this PR. The docs also build without the `sphinxext` python 2 code, so I'd say we can remove those as well.

Comment (Paulnkk) [reply doc/sphinxext/docscrape.py:472]: @phschiele I removed the files related to the nba ranking example, but when I create the html files in `doc/build/html`, references related to the docs appear again, I am sorry but not sure how to deal with this

Comment (phschiele) [reply doc/sphinxext/docscrape.py:472]: I can give it a try and push on this branch if it works.

**Review by phschiele — APPROVED** [review]
Lets merge this PR. The `docscrape.py` file is likely unused, but we can tackle that in different PR.

**Review by SteveDiamond — APPROVED** [review]

**Comment by SteveDiamond** [comment]
> Lets merge this PR. The `docscrape.py` file is likely unused, but we can tackle that in different PR.

Sounds good.

**Pull request merged**